Android Build Fail

"@merryjs/photo-viewer": "^2.2.2",
    "react": "16.6.3",
    "react-native": "0.58.4"

````
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':@merryjs_photo-viewer:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:7: error: resource android:attr/colorError not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:11: error: resource android:attr/colorError not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values/values.xml:2747: error: resource android:attr/fontStyle not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values/values.xml:2748: error: resource android:attr/font not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values/values.xml:2749: error: resource android:attr/fontWeight not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values/values.xml:2750: error: resource android:attr/fontVariationSettings not found.
  /myApp/node_modules/@merryjs/photo-viewer/android/build/intermediates/res/merged/release/values/values.xml:2751: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
````

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -11,12 +11,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-	compileSdkVersion 25
-	buildToolsVersion "25.0.3"
+	compileSdkVersion 28
+	buildToolsVersion "28.0.3"
 
 	defaultConfig {
 		minSdkVersion 16
-		targetSdkVersion 25
+		targetSdkVersion 28
 		versionCode 1
 		versionName "1.0"
 	}
@@ -32,20 +32,20 @@ repositories {
 
 
 ext {
-	supportVersion = '25.1.1'
-	frescoVersion = '1.3.0'
+	supportVersion = '28.0.3'
+	frescoVersion = '1.13.0'
 }
 
 dependencies {
 
-	compile "com.facebook.react:react-native:+"  // From node_modules
+	implementation "com.facebook.react:react-native:+"  // From node_modules
 	// compile "com.android.support:design:$supportVersion"
 	// compile "com.android.support:appcompat-v7:$supportVersion"
 	// compile "com.android.support:support-v4:$supportVersion"
 	// compile "com.android.support:cardview-v7:$supportVersion"
-	compile 'me.relex:photodraweeview:1.1.0'
+	implementation 'me.relex:photodraweeview:1.1.0'
 
-	compile "com.facebook.fresco:fresco:$frescoVersion"
-	compile "com.facebook.fresco:animated-gif:$frescoVersion"
+	implementation "com.facebook.fresco:fresco:$frescoVersion"
+	implementation "com.facebook.fresco:animated-gif:$frescoVersion"
 
 }

